tests/programs/custom/hpmcounter_basic_nostall_test/hpmcounter_basic_nostall_test.c

```diff
@@ -118,8 +118,7 @@ int main(int argc, char *argv[])
   err_cnt += chck(minstret, 4);
   printf("\nCycle count while running = %d", count);
   printf("\nMCYCLE counted cycles = %d\n", mcycle_count);
-  err_cnt += chck(count, mcycle_count);
-  err_cnt += chck(count, 5);
+  err_cnt += chck(mcycle_count, 5);
```

Comment:
This change looks odd, I need to take a closer look at this

Reply:
I double checked, this and the change in the next test (hpmcounter_basic_test.c) should not be included. S does not have the same counters as X, so this check should remain as it was.

Reply:
Okei, thank you! I will fix it.

Reply:
RESOLVED